Shared/Common/Extensions/ASCredentialIdentityStore+.swift

```diff
@@ -0,0 +1,16 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Foundation
+import AuthenticationServices
+
+@available(iOS 12, *)
+protocol CredentialIdentityStoreProtocol {
+    func getState(_ completion: @escaping (ASCredentialIdentityStoreState) -> Void)
+    func removeAllCredentialIdentities(_ completion: ((Bool, Error?) -> Void)?)
```

Comment:
Suggest using a typealias here.
e.g. 

```swift
typealias CredentialCompletion = ((Bool, Error?) -> Void)
```

Then use it as optional in the method parameter.

```swift
func removeAllCredentialIdentities(_ completion: CredentialCompletion?)
```

Reply:
doing this messes up the injection pattern that these are useful for :(